src/App.vue

```diff
@@ -312,11 +312,13 @@
       </div>
     </v-main>
   </v-app>
-  <About v-if="showAboutDialog" @update:show-about-dialog="showAboutDialog = $event" />
-  <Tutorial :show-tutorial="interfaceStore.isTutorialVisible" />
   <VideoLibraryModal :open-modal="interfaceStore.isVideoLibraryVisible" />
-  <VehicleDiscoveryDialog v-model="showDiscoveryDialog" show-auto-search-option />
-  <UpdateNotification v-if="isElectron()" />
+  <About v-if="showAboutDialog" @update:show-about-dialog="showAboutDialog = $event" />
```

Comment:
This line is removed from 315 (before the `VideoLibraryModal` component) and re-added on 316 (after the `VideoLibraryModal` component). Why was that??